Sources/NostrSDK/EventKind.swift

```diff
@@ -69,6 +74,7 @@ public enum EventKind: RawRepresentable, CaseIterable, Codable, Equatable {
         case .textNote: return 1
         case .recommendServer: return 2
         case .contactList: return 3
+        case .directMessage: return 4
```

Comment:
Add it also to `allCases` above. For some reason GitHub wouldn't let me comment there.